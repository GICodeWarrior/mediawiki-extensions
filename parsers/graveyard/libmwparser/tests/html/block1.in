<ul>
<li><div>text</li>
</ul>