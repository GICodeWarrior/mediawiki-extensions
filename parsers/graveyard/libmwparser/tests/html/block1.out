BEGIN ARTICLE
    BEGIN BULLET LIST
        BEGIN BULLET LIST ITEM
            BEGIN DIV
                WORD[text]
                SPECIAL[</]
                WORD[li]
                SPECIAL[>]
                NEWLINE
                SPECIAL[</]
                WORD[ul]
                SPECIAL[>]
            END DIV
        END BULLET LIST ITEM
    END BULLET LIST
END ARTICLE
